test/integration/suites/delegatedidentity/docker-compose.yaml

```diff
@@ -13,3 +13,4 @@ services:
     volumes:
       - ./conf/agent:/opt/spire/conf/agent
     command: ["-config", "/opt/spire/conf/agent/agent.conf"]
+    user: 0:0
```

Comment:
Thank you for bringing this up. The test is currently running the `delegatedidentityclient` process with `docker-compose exec -u 1001` and `docker-compose exec -u 1002` to have different selectors discovered in each case. Since a gid is not specified, 0 is used, so this effectively means that this is executed as 1001:0 and 1002:0 respectively. The admin API has 0770 permission bits in the exposed socket, so it requires to match the group. I'll have this fixed to not use root.